crates/chat-cli/src/cli/chat/mcp.rs

```diff
@@ -256,7 +256,7 @@ fn resolve_scope_profile(ctx: &Context, scope: Option<Scope>) -> Result<PathBuf>
 
 fn expand_path(ctx: &Context, p: &str) -> Result<PathBuf> {
     let p = shellexpand::tilde(p);
-    let mut path = PathBuf::from(p.as_ref());
+    let mut path = PathBuf::from(p.as_ref() as &str);
```

Comment:
Adding typed-path to support cross OS support cause `as_ref` to become ambiguous and explicit casts need to be added.